tests/datasource/fluent/conftest.py

```diff
@@ -29,9 +46,30 @@
 )
 from tests.sqlalchemy_test_doubles import Dialect, MockSaEngine
 
+if TYPE_CHECKING:
+    from pytest import FixtureRequest
+    from requests import PreparedRequest
+
+    from great_expectations.data_context import CloudDataContext
+
+
 FLUENT_DATASOURCE_TEST_DIR: Final = pathlib.Path(__file__).parent
 PG_CONFIG_YAML_FILE: Final = FLUENT_DATASOURCE_TEST_DIR / FileDataContext.GX_YML
 
+
+GX_CLOUD_MOCK_BASE_URL: Final[str] = "https://app.greatexpectations.fake.io"
```

Comment:
My preference would be to move gx cloud specific fixtures and helpers into their own file. This isn't blocking though.

Reply:
Yeah if it's okay I'll make these changes in followup.